web_app_drawer/static/src/less/app_drawer.less

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.app-drawer-nav {
+    border-color: @dropdown-border;
+    background-color: @dropdown-bg;
+    border: 1px solid @dropdown-fallback-border; // IE8 fallback
+    border: 1px solid @dropdown-border;
+    -webkit-border-radius: @border-radius-base;
+    -moz-border-radius: @border-radius-base;
+    border-radius: @border-radius-base;
+    .box-shadow(0 6px 12px rgba(0, 0, 0, .175));
+    background-clip: padding-box;
+}
+
+.app-drawer-title {
+    float: none;
+}
+
+.app-drawer-icon-app {
+    height: @app-drawer-icon-size;
+    width: @app-drawer-icon-size;
+    margin: @app-drawer-icon-margin;
+}
+
+.drawer-nav {
+    width: @app-drawer-width;
+    z-index: 9999;
+}
+
+.drawer--left .drawer-nav {
+    left: -@app-drawer-width;
+}
+
+.drawer--left.drawer-open .drawer-hamburger {
+    left: @app-drawer-width;
+}
+
+.drawer--right .drawer-nav {
+    right: -@app-drawer-width;
+}
+
+.drawer-open .oe-right-toolbar {
+    display: none;
+}
+
+.drawer-closed .oe-right-toolbar {
+    display: block;
+}
+
+#appDrawerAppPanelHead {
+    height: @app-drawer-title-height;
+}
+
+/* App Drawer Toggle */
+
+.app-drawer-toggle {
+    background-color: transparent;
+}
+
+.drawer .app-drawer-icon-close {
+    display: none;
+    height: app-drawer-title-height;
+}
+
+.drawer--left.drawer-open .app-drawer-icon-close {
+    display: block;
+    position: fixed;
+    z-index: 9000;
+    left: @app-drawer-width;
+    top: 0;
+}
+
+.drawer--right.drawer-open .app-drawer-icon-close {
+    display: block;
+    position: fixed;
+    z-index: 9000;
+    left: -@app-drawer-width;
+    top: 0;
+}
```

Comment:
If you do https://github.com/OCA/web/pull/396/files#r76194991, you will probably be able to remove from line 61 to here.